Internal: Documentation of `document/dynamic` component. [ED-5574] (#16981)
elementor/elementor: docs/assets/dev/js/editor/document/component.md

  ![legend-section-widget-tabs](_images/legend-section-widget-tabs.png)

> Every element in the document has a unique ID, which is used to identify it in the editor, every element has Unique [Container](#) Object, the container object
> can be found by using the following code:

Reviewer: ```suggestion
Every element in the document has a unique ID which is used to identify it in the editor, and is resolved to a unique [Container](#) object that can be accessed using the following method:
```

```javascript
elementor.getContainer( ID );
```
The next example/documents will describe the following parameters:

| Property     | Type                  | Description |
|---           |---                    |---|
| _container_  | `{Container}`         | Target.
| _containers_ | `{Container[]}`       | Targets.

> The command(s) can accept container or containers not both.

Reviewer: I don't understand the context here
